contrib/epee/include/epee/net/abstract_tcp_server2.inl

```diff
@@ -141,8 +141,8 @@ PRAGMA_WARNING_DISABLE_VS(4355)
 
     boost::system::error_code ec;
     auto remote_ep = socket().remote_endpoint(ec);
-    CHECK_AND_NO_ASSERT_MES(!ec, false, "Failed to get remote endpoint: " << ec.message() << ':' << ec.value());
-    CHECK_AND_NO_ASSERT_MES(remote_ep.address().is_v4() || remote_ep.address().is_v6(), false, "only IPv4 and IPv6 supported here");
+    //CHECK_AND_NO_ASSERT_MES(!ec, false, "Failed to get remote endpoint: " << ec.message() << ':' << ec.value());
+    //CHECK_AND_NO_ASSERT_MES(remote_ep.address().is_v4() || remote_ep.address().is_v6(), false, "only IPv4 and IPv6 supported here");
```

Comment:
Commenting these out might cause issues, because the originals are something like `if (condition) { ...log stuff...; return false }`, but commented out there won't be the early return on failure conditions.